es/lessons/basics/collections.md

```diff
@@ -68,7 +72,7 @@ iex> tl [3.14, :pie, "Apple"]
 [:pie, "Apple"]
 ```
 
-Además de la funciones citadas, puedes usar el operador tubería `|`; veremos este patrón en futuras lecciones:
+Además de la funciones citadas, puedes usar [coincidencia de patrones](../pattern-matching/) y el operador tubería `|` para dividir una lista en cabeza y cola; veremos este patrón en futuras lecciones:
```

Comment:
I think we are mixing terminology here: "operador tubería" literally translates back to English as "pipe operator", which is `|>`. We are talking about the "cons operator" `|` here. We should call it "operador cons" for now. I will investigate what's called in Spanish by other authors, and get back here later on.